[2025-01-23T13:31:22.907+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: task45_train_models_v2.task234_training_models.task_model2 manual__2025-01-23T13:31:09.673757+00:00 [queued]>
[2025-01-23T13:31:22.943+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: task45_train_models_v2.task234_training_models.task_model2 manual__2025-01-23T13:31:09.673757+00:00 [queued]>
[2025-01-23T13:31:22.944+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-01-23T13:31:23.018+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): task234_training_models.task_model2> on 2025-01-23 13:31:09.673757+00:00
[2025-01-23T13:31:23.045+0000] {standard_task_runner.py:60} INFO - Started process 8022 to run task
[2025-01-23T13:31:23.053+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'task45_train_models_v2', 'task234_training_models.task_model2', 'manual__2025-01-23T13:31:09.673757+00:00', '--job-id', '121', '--raw', '--subdir', 'DAGS_FOLDER/task45_train_models_v2.py', '--cfg-path', '/tmp/tmp480xjk2y']
[2025-01-23T13:31:23.068+0000] {standard_task_runner.py:88} INFO - Job 121: Subtask task234_training_models.task_model2
[2025-01-23T13:31:23.133+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2025-01-23T13:31:23.237+0000] {task_command.py:423} INFO - Running <TaskInstance: task45_train_models_v2.task234_training_models.task_model2 manual__2025-01-23T13:31:09.673757+00:00 [running]> on host a8cdec4ae828
[2025-01-23T13:31:23.529+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='task45_train_models_v2' AIRFLOW_CTX_TASK_ID='task234_training_models.task_model2' AIRFLOW_CTX_EXECUTION_DATE='2025-01-23T13:31:09.673757+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-23T13:31:09.673757+00:00'
[2025-01-23T13:31:23.847+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-23T13:31:23.886+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=task45_train_models_v2, task_id=task234_training_models.task_model2, execution_date=20250123T133109, start_date=20250123T133122, end_date=20250123T133123
[2025-01-23T13:31:24.111+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-23T13:31:24.256+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
